charts/kong/Chart.yaml

```diff
@@ -10,5 +10,5 @@ maintainers:
   email: harry@konghq.com
 name: kong
 sources:
-version: 1.1.1
+version: 1.1.2
```

Comment:
Please remove the version upgrade.
We are drafting contributing policies for this repo right now.
CI failures for this can be ignored. We are also working on fixing the CI as well.

Reply:
Right, will do.